tsconfig.json

```diff
@@ -18,15 +18,22 @@
 			"ES2020",
 			"ESNext"
 		],
-		"module": "ESNEXT",
-		"moduleResolution": "node",
+		"module": "preserve",
+		"moduleResolution": "bundler",
 		"noEmit": true,
 		"noUnusedLocals": true,
 		"resolveJsonModule": true,
 		"skipLibCheck": true,
 		"strict": true,
 		"target": "ESNEXT",
-		"noUncheckedIndexedAccess": true
+		"noUncheckedIndexedAccess": true,
+		"customConditions": ["development"]
```

Comment:
This tells typescript to use the `development` field defined in package.json `exports` which for `@guardian/commercial` points to the source typescript.